pom.xml

```diff
@@ -74,8 +74,8 @@
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-compiler-plugin</artifactId>
 				<configuration>
-					<source>1.6</source>
-					<target>1.6</target>
+					<source>1.7</source>
```

Comment:
Both need to be bumped to 1.8 it seems.

Reply:
It's weird, they were already in different versions, but I guess that makes sense